core/mavlink_commands.h

```diff
@@ -24,6 +24,7 @@ class MAVLinkCommands {
         COMMAND_DENIED,
         TIMEOUT,
         IN_PROGRESS,
+        NO_SETPOINT_SET,
```

Comment:
Nono, a MAVLink command can't ever have that result. It's an invention of the SDK :)

Reply:
Ah I see. OK I removed from mavlink command results and `offboard_result_from_command_result()`.